### ISSUE TITLE
[Introspection] `-` is not accepted as character in column name

### ISSUE THREAD

**Issue by janpio** [prisma/prisma2#1079]
```
error: Unexpected token. Expected one of: field type.
  -->  schema.prisma:86
   | 
85 |   device_id Int    @id
86 |   F23-      Float?
   |      ^ Unexpected token.
87 |   F24-26    Float?
   |
```
(https://github.com/prisma/introspection-engine-output/pull/5/files#diff-628e213e8d852904e07ea4d461bd1796) 

from model:
```
model sample_submission {
  device_id Int    @id
  F23-      Float?
  F24-26    Float?
  F27-28    Float?
  F29-32    Float?
  F33-42    Float?
  F43+      Float?
  M22-      Float?
  M23-26    Float?
  M27-28    Float?
  M29-31    Float?
  M32-38    Float?
  M39+      Float?
}
```
(https://github.com/prisma/introspection-engine-output/pull/5/files#diff-94e52fe6c83734157934ac39f3f3daaaR86-R100)

original DB:

```
CREATE TABLE `sample_submission` (
	`device_id` BIGINT(20) NOT NULL,
	`F23-` FLOAT NULL DEFAULT NULL,
	`F24-26` FLOAT NULL DEFAULT NULL,
	`F27-28` FLOAT NULL DEFAULT NULL,
	`F29-32` FLOAT NULL DEFAULT NULL,
	`F33-42` FLOAT NULL DEFAULT NULL,
	`F43+` FLOAT NULL DEFAULT NULL,
	`M22-` FLOAT NULL DEFAULT NULL,
	`M23-26` FLOAT NULL DEFAULT NULL,
	`M27-28` FLOAT NULL DEFAULT NULL,
	`M29-31` FLOAT NULL DEFAULT NULL,
	`M32-38` FLOAT NULL DEFAULT NULL,
	`M39+` FLOAT NULL DEFAULT NULL,
	PRIMARY KEY (`device_id`)
)
COLLATE='utf8_general_ci'
ENGINE=InnoDB
;
```

![image](https://user-images.githubusercontent.com/183673/70396467-85339700-1a09-11ea-8488-47c02c35126f.png)

**Comment by pantharshit00** [comment]
I can confirm this bug. A temporary workaround here would be to add the @map directive to those fields and changing the identifier to a valid name.

**Comment by janpio** [comment]
fixed via #1139
